pyproject.toml

```diff
@@ -98,6 +106,7 @@ cache-dir = '.cache/ruff'
 [tool.ruff.per-file-ignores]
 '__init__.py' = [
     'F401',  # * imported but unused
+    'I001',  # Import block is un-sorted or un-formatted
```

Comment:
It will only ignore the sorting in the `__init__.py` files. I also tried to use `noqa` inside `audbackend/__init__.py` as I do in `audbackend/core/api.py`, but for some reason it did not work.

Reply:
Ok, then it's fine. Maybe align the formatting of the two comments?

Reply:
Usually I copy the message you get back from `ruff` as a comment there.
The `*` stays for arbritray text as it would return:

```
audbackend/__init__.py:1:33: F401 [*] `audbackend.core.api.access` imported but unused
audbackend/__init__.py:2:33: F401 [*] `audbackend.core.api.available` imported but unused
audbackend/__init__.py:3:33: F401 [*] `audbackend.core.api.create` imported but unused
...
```

Or are you referring to something else?

Reply:
Ok, I didn't know that. Makes sense.